src/index.js

```diff
@@ -13,7 +13,10 @@ const pad = (n, width, z) => {
  */
 const getChecksum = data => {
   let checksum
-  const idx1 = data.indexOf('$GP')
+  let idx1 = data.indexOf('$GP')
```

Comment:
Change two check index of `idx1` with this plese
```suggestion
  const idx1 = data.search(/^\$G(P|N)/)
```